specification/app/resource-manager/readme.md

```diff
@@ -53,6 +55,22 @@ input-file:
   - Microsoft.App/preview/2023-05-02-preview/ManagedEnvironmentsDaprComponents.json
   - Microsoft.App/preview/2023-05-02-preview/ManagedEnvironmentsStorages.json
   - Microsoft.App/preview/2023-05-02-preview/SourceControls.json
+directive:
+  - suppress: OperationIdNounVerb
```

Comment:
@mentat9 can you help review this? beacause "SuppressionReviewRequired" label is added, thanks

Reply:
Links to reasoning for the suppressions: 

- https://github.com/Azure/azure-rest-api-specs/pull/23697#issuecomment-1545915040
- https://github.com/Azure/azure-rest-api-specs-pr/pull/10719#issuecomment-1385705675

Reply:
@pauld-msft , If you want to merge this pr asap, I think you can ask ArmReview Board to review this suppression. After "Approved Suppression" label is added , I think this pr is ok to merge

Reply:
@pauld-msft @kazrael2119  approved and signed off